toponetx/algorithms/spectrum.py

```diff
@@ -118,7 +118,7 @@ def set_hodge_laplacian_eigenvector_attrs(
     --------
     >>> from toponetx import SimplicialComplex
     >>> SC=SimplicialComplex([[1,2,3],[2,3,5],[0,1]])
-    >>> SC = set_hodge_laplacian_eigenvector_attrs(SC,1,2,"down")
+    >>> set_hodge_laplacian_eigenvector_attrs(SC,1,2,"down")
```

Comment:
Try to respect coding style guidelines in the docstrings as well, e.g. we need spaces after `,` here.